### ISSUE TITLE
TCPFormer 消融实验

### ISSUE THREAD

**Issue by Spring-first** [AsukaCamellia/TCPFormer#22]
作者您好，最近我复现了您的这个方法，整个方法的性能表现和巧妙设计令人映像深刻，但是我发现了一些令人困惑的点：

1.整个模型中没有添加时序位置编码，模型全靠特征的内容区分帧，您有做过添加时序编码的消融实验吗？

2.在MemoryInducedBlock模块中，有6个layer_scale参数：

self.layer_scale =[

nn.Parameter(layer_scale_init_value * torch.ones(dim), requires_grad=True),

nn.Parameter(layer_scale_init_value * torch.ones(dim), requires_grad=True),

nn.Parameter(layer_scale_init_value * torch.ones(dim), requires_grad=True),

nn.Parameter(layer_scale_init_value * torch.ones(dim), requires_grad=True),

nn.Parameter(layer_scale_init_value * torch.ones(dim), requires_grad=True),

nn.Parameter(layer_scale_init_value * torch.ones(dim), requires_grad=True)

]
尽管这些参数在注册时使用的是可训练参数，但实际上这些参数依旧是不可训练的，这些参数的值始终是layer_scale_init_value=1e-5，这是有意为之还是作者谬误？如果有意为之是否有做过消融实验？
